crates/egui/src/response.rs

```diff
@@ -571,6 +571,10 @@ impl Response {
             return false;
         }
 
+        if self.ctx.memory(|m| m.any_popup_open()) {
+            return false;
+        }
```

Comment:
Won't this break tooltips for items inside of tooltips? I hovering one of the elements in the ComboBox dropdown list?

Reply:
oh you're probably right — testing.

Reply:
Fixed